ui/pages/lock.tsx

```diff
@@ -416,11 +416,7 @@ export default function Lock() {
                           NumberType.number
                         ),
                         time: Date.parse(lockTime.formatted),
-                        lockTime: Date.parse(
-                          hasLock && lockTime?.orig
-                            ? lockTime.orig.formatted
-                            : new Date()
-                        ),
+                        lockTime: Date.parse(new Date().toString()),
```

Comment:
I really like how much more simple this is, is this the agreed logic that the new lock is considered as being taken from today and the initial lock date is "forgotten"?

Reply:
@johnmark13 Yes:  https://discord.com/channels/690584551239581708/1006193778345914378/1021708067211055154